Cake.Recipe/Content/toolsettings.cake

```diff
@@ -57,7 +58,9 @@ public static class ToolSettings
         string gitReleaseManagerGlobalTool = "#tool dotnet:?package=GitReleaseManager.Tool&version=0.11.0",
         string gitVersionGlobalTool = "#tool dotnet:?package=GitVersion.Tool&version=5.3.4",
         string reportGeneratorGlobalTool = "#tool dotnet:?package=dotnet-reportgenerator-globaltool&version=4.6.1",
-        string wyamGlobalTool = "#tool dotnet:?package=Wyam.Tool&version=2.2.9"
+        string wyamGlobalTool = "#tool dotnet:?package=Wyam.Tool&version=2.2.9",
+        // This is using an unofficial build of kudusync so that we can have a .Net Global tool version.  This was generated from this PR: https://github.com/projectkudu/KuduSync.NET/pull/27
+        string kuduSyncGlobalTool = "#tool dotnet:https://www.myget.org/F/cake-contrib/api/v3/index.json?package=KuduSync.Tool&version=1.5.4-gc5cc5a2a19"
```

Comment:
Just reiterating what I said on gitter (in a different format :smile:):

> I am a bit uncertain about using a myget.org URL for the unofficial build, mainly due to retention policies that may cause that specific version to suddenly disappear.

Reply:
This package version is pinned, so it shouldn’t be cleaned up by retention policies, so we should be able to take this dependency without any concerns. The KuduSync tool isn’t one that has changed recently, so I don’t foresee us pushing multiple package versions.